spec/spec.md

```diff
@@ -328,15 +328,20 @@ where:
 ```
 {
     “auth_token”: “<auth_token>”,
-    “addresses”: [“<address>", ...]
+    “labelled_addresses”: [
```

Comment:
WDYT about naming this field `accounts`? Then you'd have, e.g., `accounts[0].address` and `accounts[0].label`, which captures the semantics of the array and object pretty well.

Reply:
Definitely. I've simply given up on trying to choose the right name, in favor of word salad-ing into a PR to let you come up with better ones :)